src/backend/joanie/demo/management/commands/create_dev_demo.py

```diff
@@ -624,3 +625,27 @@ def handle(self, *args, **options):  # pylint: disable=too-many-locals,too-many-
         )
 
         self.stdout.write(self.style.SUCCESS("Successfully fake data creation"))
+
+        for order in models.Order.objects.all():
+            try:
+                order.generate_schedule()
+            except ValidationError:
+                continue
+
+            if order.state == enums.ORDER_STATE_COMPLETED:
+                for installment in order.payment_schedule:
+                    order.set_installment_paid(installment["id"])
+
+            if order.state == enums.ORDER_STATE_PENDING_PAYMENT:
+                order.set_installment_paid(order.payment_schedule[0]["id"])
+
+            if order.state == enums.ORDER_STATE_FAILED_PAYMENT:
+                order.set_installment_refused(order.payment_schedule[0]["id"])
+
+            if order.state == enums.ORDER_STATE_CANCELED:
+                order.cancel_remaining_installments()
+
```

Comment:
Will there be an order with the state "refunding" (`ORDER_STATE_REFUNDING`)... if so, maybe we could add an object reflecting this state ?